### ISSUE TITLE
Fix JS

### ISSUE THREAD

**Issue by camsinfiniteram** [MichaelDeng03/BuckyBoo#1]
During initial frontend development, JS experienced significant issues, especially after the implementation of sunset-themed buttons by a peer developer. Would be best to redo using original concept but without hindering program functionality

**Comment by MichaelDeng03** [comment]
yeah make sure to include js functionality when working on the frontend. Make sure that function calls are happening, and use default data in app.py to make sure nothing is wonky in the js and the html. The chatbox is probably the hardest thing to program with the design we have rn because it has to integrate with an actual chatbot. Make sure that you're saving chat history for each user somewhere.
